modules/Gene2Phenotype.py

```diff
@@ -60,12 +54,11 @@ def process_g2p(self):
         self.write_evidence_strings(G2P_EVIDENCE_FILENAME)
 
     def generate_evidence_strings(self, filename):
+
         total_efo = 0
-        blob = self.bucket.get_blob(filename)
-        with open('/tmp/%s'%(G2P_FILENAME), 'wb') as file_obj:
-            blob.download_to_file(file_obj)
 
-        with gzip.open('/tmp/%s'%(G2P_FILENAME), mode='rt') as zf:
+        with gzip.open(self.get_gcs_filename(filename), mode='rt') as zf:
```

Comment:
it's unnecessary to call the google cloud storage if the link is public. 
instead of creating an helper function, you should use something like:
```python
with requests.get(url, stream=True) as r:
     for row in csv.DictReader(r.iter_lines(decode_unicode=True)):
          print(row)
```